Themes/default/languages/Help.english.php

```diff
@@ -623,4 +623,6 @@
 
 $helptxt['enable_sm_stats'] = 'If enabled, this will allow Simple Machines to visit your site once a month to collect basic statistics. This will help us make decisions as to which configurations to optimize the software for. For more information please visit our <a href="https://www.simplemachines.org/about/stats.php" target="_blank">info page</a>.';
 
+$helptxt['field_reg_require'] = 'When the field get require, than is this field also mandatory on profile changes.';
```

Comment:
I believe it's not properly worded; should be something like: "If the field is required on registration, it will also be required on profile changes.."